etcdctl/README.md

```diff
@@ -160,9 +160,24 @@ The protobuf encoding of the DeleteRange [RPC response][etcdrpc].
 ```bash
 ./etcdctl put foo bar
 # OK
+./etcdctl put foo1 bar
```

Comment:
@heyitsanthony sure will make it as separate example. Also do you want me to add example for prefix as well?